modOdyssey.bas

```diff
@@ -373,7 +384,7 @@ Sub DisplayRepair()
             C = QBColor(11)
             A = 0
             frmMain.lblMenu(56).Visible = False
-        ElseIf ExamineBit(Object(D).flags, 0) = 255 Or Object(D).SellPrice = 0 Then
+        ElseIf ExamineBit(Object(D).flags, 0) = 255 Then

```

Comment:
Was this preventing repair on objects that should allow it?

Reply:
Or was SellPrice somehow identifying a completely broken item?

Reply:
The object could not be repaired if sell price was set to 0 as the object repair cost was calculated from it. I changed it so that repair cost is now calculated using 500+.sellprice instead. If you think 500 gold as a minimum repair cost for all items is too much then it can be changed

Reply:
2500* not 500. However 500 May have been a better option